src/test/java/WebDriverSeleniumTests/GoogleSearchForWikiEdge.java

```diff
@@ -0,0 +1,91 @@
+package WebDriverSeleniumTests;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class GoogleSearchForWikiEdge {
+
+    @Test
+    public void GoogleSearchForWikiIsNotEmpty() throws IOException {
+
+        final Logger logger = Logger.getGlobal();
+
+
+        System.setProperty("webdriver.edge.driver", "D:\\Tools\\msedgedriver.exe");
+
+        WebDriver driver = new EdgeDriver();
+        driver.manage().window().maximize();
+
+
+        driver.get("http://google.com");
+        new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(By.name("q")));
+        driver.manage().window().maximize();
+
+        WebElement googleSearchField = new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated(By.name("q")));
```

Comment:
you can combine line 35 and 38, they are almost equals

Reply:
Why so?
In 35 I'm working with field, in 38 - with button

Reply:
Waiter removed